### PR TITLE
Accessibility: DataGrid - Fix non-contrasting column headers

### PR DESCRIPTION
**Pull request by markallenramirez**

### CODE REVIEW

**Review by williamvinogradov — DISMISSED** [review]

File: packages/devextreme/testing/testcafe/tests/dataGrid/columnHeaders.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { a11yCheck } from '../../helpers/accessibilityUtils';
+import url from '../../helpers/getPageUrl';
+import createWidget from '../../helpers/createWidget';
+
+fixture.disablePageReloads`Column Headers`
+  .page(url(__dirname, '../container.html'));
+
+test('Checking column headers via aXe', async (t) => {
+  await a11yCheck(t);
```
Comment (Raushen) [inline packages/devextreme/testing/testcafe/tests/dataGrid/columnHeaders.ts]: You need to ensure the method performs contrast checks. Now they are excluded.

**Review by Raushen — CHANGES_REQUESTED** [review]

File: packages/devextreme/testing/testcafe/tests/dataGrid/columnHeaders.ts (hunk 1 above)
Comment (Alyar666) [reply packages/devextreme/testing/testcafe/tests/dataGrid/columnHeaders.ts]: I suggest to add test for material theme.

File: packages/devextreme/scss/widgets/generic/gridBase/_colors.scss [hunk 2]
```diff
@@ -464,7 +464,7 @@ $datagrid-text-stub-background-image-path: null !default;
 }
 
 @if $color == "light" {
-  $datagrid-columnchooser-item-color: lighten($datagrid-base-color, 38.5%) !default;
+  $datagrid-columnchooser-item-color: lighten($datagrid-base-color, 26%) !default;
```
Comment (Raushen) [inline packages/devextreme/scss/widgets/generic/gridBase/_colors.scss:467]: Let's improve other themes with percentages ("dark", "darkviolet", "greenmist") and cover this cases by matrix tests.

**Review by Raushen — CHANGES_REQUESTED** [review]

File: packages/devextreme/scss/widgets/generic/gridBase/_colors.scss (hunk 2 above)
Comment (markallenramirez) [reply packages/devextreme/scss/widgets/generic/gridBase/_colors.scss:467]: Hello, further research suggests that:
- 'darkviolet' doesn't have the contrast issue
- Changing the color to #757575 or using the 26% solution doesn't work for 'greenmist'
- The color is not changed to #757575, but the 26% solution works in 'dark'

**Review by Raushen — APPROVED** [review]

**Review by Shuvaevmedia — APPROVED** [review]

**Pull request merged**